WhaleSpotting/react-app/src/components/ReportSighting.tsx

```diff
@@ -0,0 +1,92 @@
+                                    value={species}
+                                    onChange={(e) => setSpecies(e.target.value)}/>
+                            </div>
+                            <div className="input-box">
+                                <label className="details">Quantity</label>
+                                <input className="input-field" type="number" placeholder="Enter quantity"
+                                    value={quantity}
+                                    onChange={(e) => setQuantity(e.target.value)}/>
+                            </div>
+                            <div className="input-box">
+                                <label className="details">Longitude</label>
+                                <input className="input-field coordinates" type="number" placeholder="Enter your longitude"
+                                    value={longitude}
+                                    onChange={(e) => setLongitude(e.target.value)}/>
+                            </div>
+                            <div className="input-box">
+                                <label className="details">Latitude</label>
+                                <input className="input-field coordinates" type="number" placeholder="Enter your latitude"
+                                    value={latitude}
+                                    onChange={(e) => setLatitude(e.target.value)}/>
+                            </div>
+                            <div className="input-box">
+                                <label className="details">Orca Pod</label>
+                                <input className="input-field" type="text" placeholder="Enter the orca pod"
+                                    value={orcaPod}
+                                    onChange={(e) => setOrcaPod(e.target.value)}/>
+                            </div>
+                            <div className="input-box">
+                                <label>Image Url</label>
+                                <input className="input-field" type="url" placeholder="Enter your image url"
+                                    value={imageUrl}
+                                    onChange={(e) => setImageUrl(e.target.value)}/>
+                            </div>
+                            <div className="input-box description">
+                                <label>Description</label>
+                                <input className="input-field" type="textarea" placeholder="Enter description"
+                                    value={description}
+                                    onChange={(e) => setDescription(e.target.value)}/>
+                            </div>
+                        </div>
+                    </div>
+                    <div className="button">
+                        <input className="input-field" type="submit" value="Submit Sighting"/>
+                    </div>
```

Comment:
```suggestion
                    <button onClick={handleSubmit} className="submit-button">
                        Submit Sighting
                    </button>
```
I would suggest moving to using a button, and that the button has the onClick event, then we can change the form to not have an action or an onSubmit. Also then change the handle submit to this:
```
const handleSubmit = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Reply:
done

Reply:
done